third_party/terraform/utils/provider_test.go.erb

```diff
@@ -419,12 +419,12 @@ func testAccPreCheck(t *testing.T) {
 		t.Fatalf("One of %s must be set for acceptance tests", strings.Join(projectEnvVars, ", "))
 	}
 
-	if v := multiEnvSearch(regionEnvVars); v != "us-central1" {
-		t.Fatalf("One of %s must be set to us-central1 for acceptance tests", strings.Join(regionEnvVars, ", "))
+	if v := multiEnvSearch(regionEnvVars); v == "" {
```

Comment:
Could you briefly describe why this needs to change? The env vars here & below are being explicitly set to `us-central1` and `us-central1a` in the TeamCity tests, so I think this check shouldn't cause issues for ci test runs.

Reply:
This was blocking me from running the test in a location that wasn't us-central1. If I want to run a test in europe-west6 for example, I can't without the change.

Reply:
I checked on the history of this and it seems like it should be fine. TLDR for future folks: this is a safety check because not everything is available in every region - but we already default to us-central1 and us-central1-a for CI, and encourage people to do the same for tests. If someone chooses a different default region it is up to them to ensure they're choosing one that supports the resources they want to create.